js/node/script/prepack.ts

```diff
@@ -11,7 +11,7 @@ function updatePackageJson() {
   const packageCommon = fs.readJSONSync(commonPackageJsonPath);
   const packageSelf = fs.readJSONSync(selfPackageJsonPath);
   const version = packageCommon.version;
-  packageSelf.dependencies['onnxruntime-common'] = `~${version}`;
```

Comment:
Wonder why unary bit-wise compliment was added originally.

Reply:
It's a version range syntax meaning "Include everything greater than a particular version in the same minor range". https://semver.npmjs.com/

This once was the default way for version specifying in very old NPM (v2 or v3). It's usually not used now in favor of "^" (Include everything that does not increment the first non-zero portion of semver)